mockgcp/generated/mockgcp/cloud/alloydb/v1beta/resources.pb.go

```diff
@@ -172,6 +172,8 @@ const (
 	DatabaseVersion_POSTGRES_15 DatabaseVersion = 3
 	// The database version is Postgres 16.
 	DatabaseVersion_POSTGRES_16 DatabaseVersion = 4
+	// The database version is Postgres 17.
```

Comment:
Just curious, how did you update these files? Asking because git.versions file is not updated but the .pb.go files are.

Reply:
I did change the git.versions file locally and ran `./dev/tasks/update-mockgcp`. I just only kept the files that were updated in the AlloyDB folder cause I felt like updating the other API's mockgcp files was out of scope of this PR. But happy to send out a separate PR if need be to update the mockgcp for all APIs